### ISSUE TITLE
[Bug] [Integration] [LaunchDarkly] : failing to ingest flags - 

### ISSUE THREAD

**Issue by sgandon** [port-labs/ocean#487]
### Ocean Framework Version

🌊 Ocean version: 0.5.9

### Integration Version

no version in the spec.yaml file !! and docker image is `ghcr.io/port-labs/port-ocean-launchdarkly:latest` so not really helpfull ! 
```
cat .port/spec.yaml
type: launchdarkly
description: launchdarkly integration for Port Ocean
docs: https://docs.getport.io/build-your-software-catalog/sync-data-to-catalog/launchdarkly
icon: Launchdarkly
features:
  - type: exporter
    section: Feature Management
    resources:
      - kind: project
      - kind: flag
      - kind: auditlog
      - kind: environment
configurations:
  - name: launchdarklyToken
    required: true
    sensitive: true
    type: string
  - name: launchdarklyHost
    required: false
    type: string
    default: https://app.launchdarkly.com
  - name: appHost
    type: url
    required: false
 ```

### Steps to reproduce

launched the helm install command for the integration and looked at the pod logs filled with errors.

### What did you expect to see?

projects/environments/flags accessible from the catalog

### What did you see instead?

only projects and environments where created, no flags where available.
the error logs in the pod are the following:
```
Request failed with status code: 422, Error: {"ok":false,"error":"invalid_request","message":"\"relations/environments\" must be string"}
```
And indeed looking at the auditlog online the relations are json objects.
```
  "relations": {
    "environments": {
      "aws-apac": {
        "_environmentName": "aws prod APAC",
        "_site": {
```

### Are you willing to submit a PR?

- [ ] I'm willing to submit a PR!

**Comment by mk-armah** [comment]
Thanks @sgandon , Please reopen the issue if concerns stiill persists.

**Comment by MPTG94** [comment]
This should have been fixed in #502, if there is still an issue, feel free to reopen and tag me, or open a new issue
